terraform/modules/daily_snapshot/service/init.sh

```diff
@@ -1,13 +1,13 @@
 #!/bin/bash
 
-set -eux 
+set -eux
 
 # Use APT specific mechanism to ensure non-interactive operation and wait for the lock
-sudo DEBIAN_FRONTEND=noninteractive apt-get -qqq --yes -o DPkg::Lock::Timeout=-1 update
-sudo DEBIAN_FRONTEND=noninteractive apt-get -qqq --yes -o DPkg::Lock::Timeout=-1 install -y ruby ruby-dev s3cmd anacron awscli
+DEBIAN_FRONTEND=noninteractive apt-get -qqq --yes -o DPkg::Lock::Timeout=30 update
+DEBIAN_FRONTEND=noninteractive apt-get -qqq --yes -o DPkg::Lock::Timeout=30 install -y ruby ruby-dev s3cmd anacron awscli
 
```

Comment:
nonblocking: you can just have one `DEBIAN_FRONTEND=noninteractive` higher up (with a comment), and use bare `apt` operations